### ISSUE TITLE
dapr init in airgap fails when default image registry is set to ghcr

### ISSUE THREAD

**Issue by shivamkm07** [dapr/cli#938]
## Expected Behavior

dapr init --from-dir <path-to-bundle-location> should load the binaries as well as images in bundle correctly always.

## Actual Behavior

In default case (when DAPR_DEFAULT_IMAGE_REGISTRY is set to "dockerhub"), it works fine. But if the user sets it to "ghcr" it fails.

## Steps to Reproduce the Problem

$ export DAPR_DEFAULT_IMAGE_REGISTRY="ghcr"
$ cd daprbundle
$ ./dapr init --from-dir .

It gives up following logs:
- - - 
  Making the jump to hyperspace...
⚠  Local bundle installation using from-dir flag is currently a preview feature.
ℹ️  Container images will be pulled from Dapr GitHub container registry
ℹ️  Installing runtime version latest
→  Extracting binaries and setting up components... 
Dapr runtime installed to /home/shivamkm07/.dapr/bin, you may run the following to add it to your path if you want to run daprd directly:
    export PATH=$PATH:/home/shivamkm07/.dapr/bin
❌  Extracting binaries and setting up components...
❌  fail to read docker image file ghcr.io/dapr/dapr:1.7.0-rc.2: open docker/ghcr.io-dapr-dapr-1.7.0-rc.2.tar.gz: no such file or directory
- - -
## Reason behind failure 

The reason it's failing is that the [installer-bundle](https://github.com/dapr/installer-bundle/releases) pulls image from dockerhub and  saves it with filename as `daprio-dapr-1.7.0-rc.2.tar.gz`(for runtime_ver=1.7.0-rc.2).
In CLI, when the default registry is `"dockerhub"`, image is set to: `daprio/dapr:1.7.0-rc.2` and it searches for filename `daprio-dapr-1.7.0-rc.2.tar.gz` and it works fine. But when the default registry is set to `"ghcr"`, it searches for filename `ghcr.io-dapr-dapr-1.7.0-rc.2.tar.gz` which it can't find and so throws error.

## Possible Solutions

1. Don't have any change in installer-bundle workflow. In CLI, set image filename to be `daprio-dapr-<runtime_ver>.tar.gz` in case of airgap mode. This would require minimal change for now. But in future, when we decide to switch to `ghcr` in installer-bundle-workflow as well, this would require change in both CLI and workflow and would be a breaking change at that point(earlier versions of CLI wouldn't be compatible with new bundles).
2. In installer-bundle-workflow, pull from ghcr instead of dockerhub and in both CLI and workflow, set the filename to be `dapr-dapr-<runtime_ver>.tar.gz` in case of airgap mode irrespective of any environement variables. This would work fine, but again would be a breaking change in case we would switch to dockerhub from ghcr in workflow in future.
3. Don't keep the owner names at all in filenames. So for example the filename for dapr image would be just `dapr-<runtime_ver>.tar.gz`. This would be consistent in future if we switch from ghcr to dockerhub or vice versa in future.

## Release Note

RELEASE NOTE: **FIX** Bug in CLI airgap installation

RELEASE NOTE:

**Comment by mukundansundar** [comment]
This is the format I think the image should be saved as ... 
https://github.com/dapr/cli#install-by-providing-a-docker-container-registry-url
```
You can install Dapr runtime by pulling docker images from a given private registry uri by using --image-registry flag.

Note: This command expects that images have been hosted like example.io/dapr/dapr:, example.io/dapr/3rdparty/redis:, example.io/dapr/3rdparty/zipkin:
```

So for the GitHub org owner as dapr since we are publishing the dapr system image, it will be `dapr-dapr-1.7.0-rc.2.tar.gz` as filename in the installer bundle. 
If we decided to later on include 3rdparty images also, 
it will follow as `dapr-3rdparty-redis-latest.tar.gz` for example.

**Comment by artursouza** [comment]
I am fine if we remove owner from filenames in installer bundle since those are useful to segregate ownership in registries and does not apply to our own mirror images. It also means that we could change the image for redis, for example, without a single line change in code since it would still be called "redis".

**Comment by shivamkm07** [comment]
@mukundansundar This can be closed now.

**Comment by mukundansundar** [comment]
closed by #945
